packages/wallets/src/evm/walletIds.ts

```diff
@@ -0,0 +1,10 @@
+export const walletIds = {
+  coinbase: "coinbaseWallet",
+  metamask: "metamask",
+  localWallet: "localWallet",
+  smartWallet: "SmartWallet",
+  safe: "Safe",
```

Comment:
should have a consistent naming pattern here? All others start with lower case, we can rename these two to `safe` and `smartWallet`